cmd/translate/main_grpc_integration_test.go

```diff
@@ -683,6 +683,42 @@ func Test_UpdateTranslation_gRPC(t *testing.T) {
 			}
 
 			assert.Equal(t, tt.expectedCode, status.Code(err))
+
+			if tt.request == happyReq {
+				matchingTranslationExistsInService(ctx, t, tt.request.ServiceId, resp)
+			}
 		})
 	}
 }
+
+// matchingTranslationExistsInService checks incoming translation is equal to translation
+// with same language returned from listTranslations.
+func matchingTranslationExistsInService(
+	ctx context.Context,
+	t *testing.T,
+	serviceID string,
+	t1 *translatev1.Translation,
```

Comment:
Please avoid the same short names with different meanings: t - *testing.T, t2 - translatev1.Translation. Do full name.

```suggestion
	translation *translatev1.Translation,
```